src/cpp/src/continuous_batching/model_runner.hpp

```diff
@@ -319,28 +336,20 @@ class ModelRunner {
             }
             sequence_group->set_output_seq_len(matmul_gathering_is_available ? output_seq_len : num_scheduled_tokens);
         }
-
-        if (sequence_group_type == SequenceGroupType::TOKENS) {
-            m_request.set_tensor("input_ids", input_ids);
```

Comment:
I can't find `input_ids` being set. I assume caching with `_get_or_resize_tensor()` handles this implicitly. The problem is that `_get_or_resize_tensor()` may not use the result of `m_request.get_tensor()`. The reader is going to assume it may happen to any tensor because `score_aggregation_window` is listed as an example. Please revert removal of `set_tensor()` calls for all such tensors